workshop_app/templates/workshop_app/workshop_status.html

```diff
@@ -0,0 +1,190 @@
+{% extends "workshop_app/base.html" %}
+
+{% block title %}
+    Workshop Status
+{% endblock %}
+
+{% block extra-dependencies %}
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+{% endblock %}
+{% block extra-custom-scripts %}
+    <script>
+        //ToolTip popup function on-hover
+        $(document).ready(function () {
+            $('[data-toggle="popover"]').popover({
+                placement: 'top',
+                trigger: 'hover'
+            });
+        });
+
+        //ToolTip popup function on-hover
+        $(document).ready(function () {
+            $('[data-toggle="popinfo"]').popover({
+                placement: 'top',
+                trigger: 'hover'
+            });
+        });
+
+        // Change date modal
+        function changeDate(date) {
+            let previous_date = new Date(date);
+            let dateToday = new Date();
+            let upto = new Date();
+
+            previous_date.setDate(previous_date.getDate() + 1);
+            upto.setFullYear(dateToday.getFullYear() + 1);
+
+            let counter = date.split(" ");
+            const id = counter.slice(-1).pop();
+            if (date[0] === 'P') {
+                counter = '.pDate' + id
+                $(counter).datepicker({
+                    changeMonth: true,
+                    changeYear: true,
+                    minDate: dateToday,
+                    maxDate: upto,
+                    dateFormat: "yy-mm-dd",
+                });
+                $(".ui-dialog-content").dialog("close");
+                $('.myDialogP' + id).dialog();
+
+            } else {
+                counter = '.rDate' + id;
+                $(counter).datepicker({
+                    changeMonth: true,
+                    changeYear: true,
+                    minDate: dateToday,
+                    maxDate: upto,
+                    dateFormat: "yy-mm-dd",
+                });
+                $(".ui-dialog-content").dialog("close");
+                $('.myDialogR' + id).dialog();
+
+            }
+        };
+
+    </script>
+{% endblock %}
+
+
+{% block content %}
+    {% if workshops %}
```

Comment:
Do we need this condition here, what goes wrong if there is no workshop?

Reply:
If there are no workshops, instead of showing a blank table, this shows a message.

Reply:
Use the ```if not ... ``` here, no point in keeping the whole HTML codse in an indented block